activesupport/lib/active_support/syntax_error_proxy.rb

```diff
@@ -12,15 +12,15 @@ def backtrace
       parse_message_for_trace + super
     end
 
-    class BacktraceLocation < Struct.new(:path, :lineno, :to_s)
+    class BacktraceLocation < Struct.new(:path, :lineno, :to_s) # :nodoc:
```

Comment:
This one surprises me a bit because `SyntaxErrorProxy` already has `:nodoc:`, but somehow the nested classes are documented: https://edgeapi.rubyonrails.org/classes/ActiveSupport/SyntaxErrorProxy/BacktraceLocation.html